tests/MongoDB.Driver.Tests/AsyncCursorTests.cs

```diff
@@ -53,6 +58,50 @@ public void Cursor_should_not_throw_exception_after_double_close([Values(false,
             }
         }
 
+        [SkippableTheory]
+        [ParameterAttributeData]
+        public void KillCursor_should_actually_work(
+            [Values(false, true)] bool async)
+        {
+            RequireServer.Check().Supports(Feature.KillCursorsCommand);
+            var eventCapturer = new EventCapturer().Capture<CommandSucceededEvent>(x => x.CommandName.Equals("killCursors"));
+            using (var client = DriverTestConfiguration.CreateDisposableClient(eventCapturer))
+            {
+                IAsyncCursor<BsonDocument> cursor;
+                var database = client.GetDatabase("test");
+                var collection = database.GetCollection<BsonDocument>(GetType().Name);
+                var documents = new List<BsonDocument>();
+                for (int i = 0; i < 1000; i++)
+                {
+                    documents.Add(new BsonDocument("x", i));
+                }
+
+                if (async)
+                {
+                    collection.InsertManyAsync(documents).GetAwaiter().GetResult();
+                    cursor = collection.FindAsync("{}").GetAwaiter().GetResult();
+                    cursor.MoveNextAsync().GetAwaiter().GetResult();
+                }
+                else
+                {
+                    collection.InsertMany(documents);
+                    cursor = collection.FindSync("{}");
+                    cursor.MoveNext();
+
+                }
+
+                long cursorId;
```

Comment:
Combine this line with the following one.  Initialization and assignment usually don't need to be separated.

Reply:
Done. I think that may have been a holdover from a previous commit where I needed to declare it separately, but it's not necessary anymore.